collector/container/rhel/final-step.sh

```diff
@@ -6,11 +6,11 @@ echo '/usr/local/lib' > /etc/ld.so.conf.d/usrlocallib.conf && ldconfig
 
 mv collector-wrapper.sh /usr/local/bin/
 chmod 700 bootstrap.sh
-dnf upgrade -y
-dnf install -y kmod
+microdnf upgrade -y
+microdnf install -y kmod findutils
 
-dnf clean all
-rpm --query --all 'curl' '*rpm*' '*dnf*' '*libsolv*' '*hawkey*' 'yum*' | xargs rpm -e --nodeps
-rm -rf /var/cache/dnf
+microdnf clean all
+rpm --query --all 'curl' '*rpm*' '*dnf*' '*libsolv*' '*hawkey*' 'yum*' 'findutils' | xargs -t rpm -e --nodeps
+rm -rf /var/cache/yum
```

Comment:
Should this be `dnf` still? or does `microdnf` use the same cache path as yum for some reason?

Reply:
I'm not 100% sure why, but the ubi-minimal image uses the `/var/cache/yum` path, guess it might be based on an earlier version of yum/dnf that hadn't change the location of the cache. You can test this easily in a container, running `microdnf update` and checking the contents of `/var/cache`.